src/Test.UnitTests.Sarif/Test.UnitTests.Sarif.csproj

```diff
@@ -23,88 +23,6 @@
-    <None Remove="TestData\PrereleaseCompatibilityTransformer\Inputs\V1.sarif" />
-    <None Remove="TestData\PrereleaseCompatibilityTransformer\Inputs\WithExternalPropertyFiles.01-24.sarif" />
-    <None Remove="TestData\PrereleaseCompatibilityTransformer\Inputs\WithSuppressions.sarif" />
-    <None Remove="TestData\RoundTripping\ExpectedOutputs\InvocationOverridesRuleDefaultConfigurationOfError.sarif" />
-    <None Remove="TestData\RoundTripping\ExpectedOutputs\NoRuleDefaultConfiguration.sarif" />
-    <None Remove="TestData\RoundTripping\ExpectedOutputs\PropertyBagComprehensiveValueTypes.sarif" />
-    <None Remove="TestData\RoundTripping\ExpectedOutputs\RuleDefaultConfigurationIsError.sarif" />
-    <None Remove="TestData\RoundTripping\ExpectedOutputs\RuleDefaultConfigurationIsWarning.sarif" />
-    <None Remove="TestData\RoundTripping\Inputs\InvocationOverridesRuleDefaultConfigurationOfError.sarif" />
-    <None Remove="TestData\RoundTripping\Inputs\NoRuleDefaultConfiguration.sarif" />
-    <None Remove="TestData\RoundTripping\Inputs\PropertyBagComprehensiveValueTypes.sarif" />
-    <None Remove="TestData\RoundTripping\Inputs\RuleDefaultConfigurationIsError.sarif" />
-    <None Remove="TestData\RoundTripping\Inputs\RuleDefaultConfigurationIsWarning.sarif" />
-    <None Remove="TestData\RoundTripping\RoundTripping.sarif" />
-    <None Remove="TestData\SarifCurrentToVersionOneVisitor\ExpectedOutputs\OneRunWithNotificationTime.sarif" />
-    <None Remove="TestData\SarifCurrentToVersionOneVisitor\Inputs\OneRunWithNotificationTime.sarif" />
-    <None Remove="TestData\SarifLogger\SimpleExample.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\MinimumWithLanguage.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\NestedInnerExceptionsInNotifications.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\NonDottedQuadFileVersion.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\OneRunWithAllReportingDescriptors.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\ResultLocationsAsEmptyArray.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\MinimumWithLanguage.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\NestedInnerExceptionsInNotifications.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\NonDottedQuadFileVersion.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\OneRunWithAllReportingDescriptors.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\ResultLocationsAsEmptyArray.sarif" />
-    <None Remove="TestData\UriConverter\UriConverterTests.json" />
-    <None Remove="TestData\V2ResultMatcher\elfie-arriba.sarif" />
-    <None Remove="xunit.runner.json" />
```

Comment:
Was it intentional or unintentional that all of these were removed from the .csproj?

Reply:
yes intentional, I had a previous PR comment from Michael that <ItemGroup>that has only <None Remove> are not needed should be removed.